low-level/files/opt/seagate/sspl/bin/conf_sensors_enable

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
can we rename to conf_based_sensor_enable

Reply:
Done